packages/tslint-config/eslint.json

```diff
@@ -2,7 +2,8 @@
   "$schema": "http://json.schemastore.org/tslint",
   "extends": [
     "./tslint.json",
-    "tslint-config-eslint/prettier"
+    "tslint-config-eslint/prettier",
+    "tslint-config-eslint/sonar"
   ],
   "rules": {
```

Comment:
These rules can be removed then.

Reply:
so, should we add this rule directly in rules, and remove prettier and sonar's extends ?

Reply:
I prefer:

```jsonc
{
  "$schema": "http://json.schemastore.org/tslint",
  "extends": [
    "./tslint.json",
    "tslint-config-eslint/sonar",
    "tslint-config-eslint/prettier"
  ]
}
```

`prettier` should always take highest priority.

Reply:
sorry i forgot the priority

Reply:
? Why we're not removing `rules`?

Reply:
updated this PR